pkgs/servers/gotosocial/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, lib
+, fetchurl
+, fetchFromGitHub
+, buildGoModule
+, nixosTests
+}:
+let
+  owner = "superseriousbusiness";
+  repo = "gotosocial";
+
+  version = "0.8.1";
+  source-hash = "sha256-nNkcK9MmhL6HbiR8yE2OI/XbBATyJ8qNIUDmJwXxD6I=";
+  web-assets-hash = "sha256-PGllF0aGI7fSF0U6MIkgeGc+Oa0eEdiG0OWuHCe67Ko=";
+
+  web-assets = fetchurl {
+    url = "https://github.com/${owner}/${repo}/releases/download/v${version}/${repo}_${version}_web-assets.tar.gz";
+    sha256 = web-assets-hash;
+  };
+in
+buildGoModule rec {
+  inherit version;
+  pname = repo;
+
+  src = fetchFromGitHub {
+    inherit owner repo;
+    rev = "refs/tags/v${version}";
+    hash = source-hash;
+  };
+
+  vendorHash = null;
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.Version=${version}"
+  ];
+
+  postInstall = ''
+    tar xf ${web-assets}
+    mkdir -p $out/share
+    mv web $out/share/
```

Comment:
The folder name `web` could cause a collision in a linkfarm/lfs. I commonly see other packages use a folder name that match the package, i.e. `$out/share/gotosocial`

Reply:
Fixed, thanks!